shared/route-tree/render-route.js

```diff
@@ -28,7 +28,6 @@ export type RouteRenderStack = I.Stack<RenderRouteResult>
 // Components rendered by routes receive the following props:
 export type RouteProps<P, S> = {
   // Whether the route is the primary onscreen route.
-  isActiveRoute: boolean,
```

Comment:
just remove this concept and pausableConnect. our mapStateToProps should just be fast